pom.xml

```diff
@@ -150,6 +154,34 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
+        <executions>
+          <execution>
+            <!-- Produce a jar with a platfrom-specific classifier. -->
+            <id>default-jar</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <phase>package</phase>
+            <configuration>
+              <classifier>${os.detected.classifier}</classifier>
+            </configuration>
+          </execution>
+          <execution>
+            <!-- Also produce a jar with 'java' classifier. This can be built on any platform. -->
```

Comment:
I thought we build the JAR without native libraries with no classifier already? Could you explain why we need this?